### DISCUSSION TITLE
How to use random prompts without repetition

### DISCUSSION THREAD

**Discussion by rawmen0x** [adieyal/comfyui-dynamicprompts#26] [Q&A]
Hi,

I'm trying to use a wildcard on Random Prompts from a list with 100 scene ideas, I noticed that some scenes repeat a lot more than others, is there a way to iterate over each without repeating?

Prompt like this:
a cartoon of a __scene__, etc

I tried:
{item1|item2...}
or line breaks:
item1
item2
...

Either way it's not working.

Thank you.

**Comment by adieyal** [comment]
Unfortunately not, each sample is independent. Have you have a look at the combinatorial node? That be useful in this case.
